.github/workflows/tox.yml

```diff
@@ -0,0 +1,30 @@
+name: Tox Test
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:20.04
+    strategy:
+      matrix:
+        python: ["3.8", "3.9", "3.10"]
```

Comment:
I added 3.9 and 3.10 but there is no actual requirements on this.

Reply:
Ubuntu 20.04 uses 3.8 by default. Ubuntu 22.04 uses 3.10. I suggest leaving it just because of that.